tina/config.tsx

```diff
@@ -61,6 +61,19 @@ const config = defineStaticConfig({
     publicFolder: "public", // The public asset folder for your framework
     outputFolder: "admin", // within the public folder
   },
+  ui: {
+    previewUrl: (context) => {
+      const { branch } = context;
+      const url =
+        branch === "main"
+          ? "https://www.ssw.com.au"
+          : `https://${process.env.NEXT_PUBLIC_APP_SERVICE_NAME}-pr-${process.env.NEXT_PUBLIC_PULL_REQUEST_ID}.azurewebsites.net/`;
```

Comment:
I think the URL should be passed and not calculated inside the app

Reply:
Done - Now I'm passing it through the pipeline, we can't pass the dynamic URL from the slot itself since we need it before creating the docker image.